realm/realm-library/src/main/cpp/io_realm_internal_Util.cpp

```diff
@@ -71,6 +84,17 @@ JNIEXPORT void JNI_OnUnload(JavaVM* vm, void*)
 
 JNIEXPORT void JNICALL Java_io_realm_internal_Util_nativeSetDebugLevel(JNIEnv*, jclass, jint level)
 {
+    /**
```

Comment:
if you add `LogLevel.java` to the cmake `create_javah`, it will generate the header fine with all trace level defined in the cpp land. And just use them instead of using immediate numbers.